corehq/apps/toggle_ui/static/toggle_ui/js/flags.js

```diff
@@ -17,7 +17,7 @@ hqDefine('toggle_ui/js/flags', [
             if (viewModel.tagFilter() === 'all') {
                 return true;
             }
-            var tag = aData[0].replace(/\n/g," ").replace(/<.*?>/g, "");
+            var tag = aData[0].replace(/\s+/g," ").replace(/<.*?>/g, "").replace(/^\d+ /, "");
```

Comment:
Is the final replace (`.replace(/^\d+ /, "")`) intended to replace more than the first occurrence/match? If yes it needs a `g` flag on the regexp.

Reply:
Nope, it's just intended to strip numbers from the beginning of the line. The numbers were introduced recently to fix sort order, which broke filtering.

I wish we did this in a cleaner way, but it wasn't my focus in this PR, just something I came across while testing.

Reply:
ha, I missed the `^` ... which makes it obvious it's only matching the first occurrence. Sorry for the noise.